mephisto/operations/operator.py

```diff
@@ -306,7 +306,7 @@ def _track_and_kill_runs(self):
                     tracked_run.architect.shutdown()
                     tracked_run.task_launcher.shutdown()
                     del self._task_runs_tracked[task_run.db_id]
-            time.sleep(2)
+            time.sleep(10)
```

Comment:
Could you add this as hardcoded constant (probably all caps) at the top of the file or in a constants file (if you are using one). Specially having a descriptive name helps someone who may want to debug later see what is readily available for tweaking.